pkg/corerp/api/v20231001preview/environment_conversion_test.go

```diff
@@ -76,8 +76,14 @@ func TestConvertVersionedToDataModel(t *testing.T) {
 					RecipeConfig: datamodel.RecipeConfigProperties{
 						Terraform: datamodel.TerraformConfigProperties{
 							Authentication: datamodel.AuthConfig{
-								Git: datamodel.GitAuthConfig{},
+								Git: datamodel.GitAuthConfig{
+									PAT: map[string]datamodel.SecretConfig{},
```

Comment:
Why is this changing as part of this PR?

Reply:
Removed this part. Bad copy paste :D